ndk/src/configuration.rs

```diff
@@ -136,7 +136,7 @@ impl Configuration {
     pub fn country(&self) -> String {
         let mut result = "  ".to_owned();
         unsafe {
-            ffi::AConfiguration_getCountry(self.ptr.as_ptr(), result.as_mut_ptr() as *mut _);
+            ffi::AConfiguration_getCountry(self.ptr.as_ptr(), result.as_mut_ptr());
```

Comment:
Fwiw besides the build error on x86 platforms (not sure how to cleanly perform the cast without tripping `trivial_casts` besides just disabling it either locally or not pulling in this PR altogether), this can also return two null-characters (https://developer.android.com/ndk/reference/group/configuration#aconfiguration_getcountry) and should realistically return an `Option` too.